.github/CODEOWNERS

```diff
@@ -301,10 +301,10 @@ packages/kbn-cypress-config @elastic/kibana-operations
 x-pack/plugins/dashboard_enhanced @elastic/kibana-presentation
 src/plugins/dashboard @elastic/kibana-presentation
 src/plugins/data @elastic/kibana-visualizations @elastic/kibana-data-discovery
-test/plugin_functional/plugins/data_search @elastic/kibana-app-services
+test/plugin_functional/plugins/data_search @elastic/kibana-core
```

Comment:
I think this should be data-discovery since it is related to search source

Reply:
Agreed, if it's related to search source it should go to us.